go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/matrixorigin/matrixone
 
-go 1.19
+go 1.20
```

Comment:
you cannot do this because go mod tidy is not working.   Just keep this as 1.19

Reply:
Ok, I'll change it back